api/src/pcapi/connectors/dms/api.py

```diff
@@ -116,6 +116,18 @@ def get_bic(self, dossier_id: int) -> Any:
         variables = {"dossierNumber": dossier_id}
         return self.execute_query(query, variables=variables)
 
+    def update_text_annotation(self, dossier_id: str, instructeur_id: str, annotation_id: str, value: str) -> Any:
+        query = self.build_query("pro/update_text_annotation")
```

Comment:
Je pense que c'est suffisamment générique pour que ce soit à la racine (ce n'est pas spécifique à la procédure pro)
```suggestion
        query = self.build_query("update_text_annotation")
```